config/packages/projection.yaml

```diff
@@ -0,0 +1,37 @@
+prooph_event_store:
```

Comment:
This file should be named something like `prooph_event_store.yaml` files like these should be named after there bundle name (without the bundle suffix and in lowercase. MonologBundle -> monolog)

Reply:
thx, I've changed it